pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -98,15 +99,16 @@ def build_random_data(
 new_linking_cols["new"] = {"cytoplasm": "ObjectNumber"}
 
 # Setup SingleCells Class
-ap = SingleCells(file_or_conn=file)
+ap = SingleCells(file_or_conn=file, object_feature='Metadata_ObjectNumber')
```

Comment:
please `black` this file

Reply:
I also notice that the default in `SingleCells` is set to just `ObjectNumber` (instead of `Metadata_ObjectNumber`). Is plain old `ObjectNumber` better than with a Metadata prefix?

Reply:
https://github.com/cytomining/pycytominer/pull/129/commits/5e5ad2b55ca16d0790b8cd23a148388da832e93c

`ObjectNumber` was the name in the new JUMP pilot data. I am guessing (not certain) this is what is output by newer versions of CellProfiler. Hence the change.